diff --git a/node_modules/feel-input-component/src/FeelInput/FeelInput.tsx b/node_modules/feel-input-component/src/FeelInput/FeelInput.tsx
index b915b46..5b43099 100644
--- a/node_modules/feel-input-component/src/FeelInput/FeelInput.tsx
+++ b/node_modules/feel-input-component/src/FeelInput/FeelInput.tsx
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import * as Monaco from "monaco-editor";
+import * as Monaco from "@kie-tooling-core/monaco-editor";
 import * as React from "react";
 import { useCallback, useEffect, useImperativeHandle, useMemo, useRef } from "react";
 import { FeelEditorService, SuggestionProvider } from "../Monaco";
diff --git a/node_modules/feel-input-component/src/Monaco/FeelConfigs.ts b/node_modules/feel-input-component/src/Monaco/FeelConfigs.ts
index 684b391..994f939 100644
--- a/node_modules/feel-input-component/src/Monaco/FeelConfigs.ts
+++ b/node_modules/feel-input-component/src/Monaco/FeelConfigs.ts
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import * as Monaco from "monaco-editor";
+import * as Monaco from "@kie-tooling-core/monaco-editor";
 
 export const MONACO_FEEL_LANGUAGE = "feel-language";
 
diff --git a/node_modules/feel-input-component/src/Monaco/FeelEditorService.ts b/node_modules/feel-input-component/src/Monaco/FeelEditorService.ts
index 6dca54a..8c99c52 100644
--- a/node_modules/feel-input-component/src/Monaco/FeelEditorService.ts
+++ b/node_modules/feel-input-component/src/Monaco/FeelEditorService.ts
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import * as Monaco from "monaco-editor";
+import * as Monaco from "@kie-tooling-core/monaco-editor";
 import {
   initializeFeelCompletionItemProvider,
   initializeMonacoTheme,
diff --git a/node_modules/feel-input-component/src/Monaco/FeelInitializer.ts b/node_modules/feel-input-component/src/Monaco/FeelInitializer.ts
index fdd9d27..2218660 100644
--- a/node_modules/feel-input-component/src/Monaco/FeelInitializer.ts
+++ b/node_modules/feel-input-component/src/Monaco/FeelInitializer.ts
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import * as Monaco from "monaco-editor";
+import * as Monaco from "@kie-tooling-core/monaco-editor";
 import "monaco-editor/dev/vs/editor/editor.main.css";
 import { MONACO_FEEL_LANGUAGE, MONACO_FEEL_THEME, feelTheme, feelDefaultSuggestions, feelTokensConfig } from ".";
 
